Classes/View Controllers/UINavigationItem+TitleSubtitle.swift

```diff
@@ -11,10 +11,29 @@ import UIKit
 extension UINavigationItem {
 
     func configure(filePath: FilePath) {
-        configure(title: filePath.current, subtitle: filePath.basePath)
+        let accessibilityLabel: String?
+        switch (filePath.current, filePath.basePath) {
+        case (.some(let current), .some(let basePath)):
+            let isCurrentAFile = filePath.fileExtension != nil
+            let currentType = isCurrentAFile ? "File" : "Folder"
+            accessibilityLabel = .localizedStringWithFormat("%@: %@, file path: %@", currentType, current, basePath)
+        case (.some(let current), .none):
+            // NOTE: This is currently unused, as we early-exit
+            // and set the navigationItem's title, but not it's titleView -
+            // and thus we can't set an accessibility label.
+            let isFile = filePath.fileExtension != nil
+            let type = isFile ? "File" : "File path"
+            accessibilityLabel = .localizedStringWithFormat("%@: %@", type, current)
+        case (.none, .some(let basePath)):
+            assert(false, "We should never have just a base path; this should always be accompanied by a current path!")
```

Comment:
Not sure if this is overkill; might want to test this somewhere else and remove the assert here, downgrading it to a comment? Or just leave it I suppose.

Reply:
Nah, good to call out, I like this